controller/token.go

```diff
@@ -42,10 +41,11 @@ type TokenController struct {
 	Configuration                LoginConfiguration
 	keycloakExternalTokenService keycloak.KeycloakExternalTokenService
 	providerConfigFactory        link.OauthProviderFactory
+	InitTenant                   func(ctx context.Context) error
```

Comment:
We don't need this, right?

Reply:
No, we don't. Let me clean this up.

Reply:
Done https://github.com/fabric8-services/fabric8-auth/pull/253/commits/035e4c8656bdf2fefc09873c4bea422f8d26cc0d